main.go

```diff
@@ -152,3 +155,12 @@ func mainError() error {
 
 	return nil
 }
+
+func homeDir() string {
+	home := os.Getenv("HOME")
+	if home == "" {
+		// Windows homedir.
+		home = os.Getenv("USERPROFILE")
+	}
+	return home
+}
```

Comment:
I believe [go 1.12 introduced a new function](https://golang.org/doc/go1.12#os) called `UserHomeDir`. Would that work here?

Reply:
Good shout, but after all, I had to drop that default for correct behaviour `inCluster`. :(